src/commcare_cloud/commands/deploy/command.py

```diff
@@ -90,17 +90,22 @@ def run(self, args, unknown_args):
             if environment.meta_config.always_deploy_formplayer:
                 deploy_component.append('formplayer')
 
+        rc = 0
         if 'commcare' in deploy_component:
             if 'formplayer' not in deploy_component:
                 _warn_no_formplayer()
-            return deploy_commcare(environment, args, unknown_args)
+            rc = deploy_commcare(environment, args, unknown_args)
         if 'formplayer' in deploy_component:
             if 'commcare' not in deploy_component:
                 if args.commcare_rev:
                     print(color_warning('--commcare-rev does not apply to a formplayer deploy and will be ignored'))
                 if args.fab_settings:
                     print(color_warning('--set does not apply to a formplayer deploy and will be ignored'))
-            return deploy_formplayer(environment, args)
+            if rc:
```

Comment:
Nit: I think this implicit check favors brevity over readability. Perhaps `rc` is guarantee enough that this is a shell-style return code, but I think it's implicit that 0 indicates success here, and it would be better to just spell it out (not everyone is familiar with the shell convention):

```
SUCCESS = 0
if rc == SUCCESS...
```